table-api/src/main/java/io/deephaven/api/agg/spec/AggSpecUnique.java

```diff
@@ -17,6 +18,12 @@
 @BuildableStyle
 public abstract class AggSpecUnique extends AggSpecBase {
 
+    public static final boolean INCLUDE_NULLS_DEFAULT = false;
```

Comment:
You should be consistent in your changes, here. You've added a new pattern for `AggSpecUnique`, but not similar specs like `AggSpecCountDistinct`.